### PR TITLE
Integrate twilio

### LINKED ISSUE #7: SMS Notification Integration using Twilio

### PR DESCRIPTION
**Pull request by pratishshr**
- Integrate twilio as a notifier
- Write test cases for twilio service
- Update example yml files

closes #7

### CODE REVIEW

File: src/services/twilio.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    let response = await sendNotification(payLoad);
+
+    logger.info('Sent notification to', response.to);
+    logger.debug('Result:', response);
+  } catch (err) {
+    logger.error('Error sending notification from twilio.', err);
+  }
+}
+
+function preparePayLoad(params) {
```
Comment (mesaugat) [inline src/services/twilio.js]: JSDoc missing

File: src/services/twilio.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  logger.debug('Notification Params:', params);
+  let payLoad = preparePayLoad(params);
+
+  try {
+    let response = await sendNotification(payLoad);
+
+    logger.info('Sent notification to', response.to);
+    logger.debug('Result:', response);
+  } catch (err) {
+    logger.error('Error sending notification from twilio.', err);
```
Comment (pratishshr) [inline src/services/twilio.js]: will do 👍

File: src/services/twilio.js [hunk 3]
```diff
@@ -1,19 +1,7 @@
+import messages from '../common/messages';
```
Comment (mesaugat) [inline src/services/twilio.js]: ↔️

File: src/utils/twilioClient.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import twilio from 'twilio';
+import config from '../config/config';
+
+const ACCOUNT_SID = config.notifications.twilio.accountSid;
+const AUTH_TOKEN = config.notifications.twilio.authToken;
```
Comment (mesaugat) [inline src/utils/twilioClient.js]: ↔️

**Comment by kabirbaidhya** [comment]
Looks all good to me :+1:

**Pull request merged**